frontned/src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,22 @@
+import Logo from './Logo';
+import Navigation from './Navigation';
+import SearchBar from './SearchBar';
+import styled from 'styled-components';
+
+const NavbarContainer = styled.div`
```

Comment:
I would recommend to use one styled to be consistent (use the one from @mui/material)